inc/Game.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <string>
+#include "../inc/Frame.hpp"
+
+class Game
+{
+    std::string name_;
+    Frame frame[12];
+public:
+    std::string getName() const;
+    void setName(const std::string &);
```

Comment:
Why not to set the game name in constructor only? Do we assume that the name will be changed? Or will we create a new object for a new game?